src/main/java/io/cdap/e2e/pages/locators/CdfControlCenterLocators.java

```diff
@@ -0,0 +1,167 @@
+
+  public static By clickOnTruncateButton() {
+    return By.xpath("//div//button[@data-cy='Truncate']");
+  }
+
+  @FindBy(how = How.XPATH, using = "//div[@class='empty-message-container']")
+  public static WebElement pipelineDeletedMessage;
+
+  @FindBy(how = How.XPATH, using = "//input[@class='search-input form-control'][@placeholder='Search']")
+  public static WebElement searchTabControlCenter;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='just-added-entities-list']//button[@class='btn btn-link']" +
+    "//*['icon-svg icon-wrench']")
+  public static WebElement setPreferencesIcon;
+
+  public static WebElement keyInputField() {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//input[@placeholder='key']"));
+  }
+
+  public static WebElement valueInputField() {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//input[@placeholder='value']"));
+  }
+
+  @FindBy(how = How.XPATH, using = "//button[@data-cy='save-prefs-btn']")
+  public static WebElement saveAndCloseButton;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='just-added-entities-list']" +
+    "//*[@class='icon-svg icon-ETLBatch entity-icon']")
+  public static WebElement dataPipelineTab;
+
+  @FindBy(how = How.XPATH, using = "//i[@class='fa fa-trash']")
+  public static WebElement deleteIconSetPreferences;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='entity-card-header dataset']" +
+    "//*[@class='icon-svg icon-datasets entity-icon']")
+  public static WebElement datasetEntityIcon;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='header']//*[@class='icon-svg icon-datasets']")
+  public static WebElement datasetEntityDetailsPage;
+
+  @FindBy(how = How.XPATH, using = "(//*[@class='icon-svg icon-trash'])[1]")
+  public static WebElement datasetDeleteIcon;
+
+  @FindBy(how = How.XPATH, using = "(//*[@class='icon-svg icon-cut'])[1]")
+  public static WebElement truncateDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//a[@class='link-to-detail']")
+  public static WebElement viewDetailsTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Dataset')]")
+  public static WebElement navigateOnViewDetailsPageDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Back')]")
+  public static WebElement backTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//*[@class='icon-svg icon-close']")
+  public static WebElement closeTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying all entities, sorted by Newest')]")
+  public static WebElement datasetApplicationsArtifactsMessage;
+
+  public static By locateDropdownListItem(String option) {
+    return By.xpath("//input[@data-cy='" + option + "']");
+  }
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Schema')]")
+  public static WebElement schemaTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Programs')]")
+  public static WebElement programsTabDatasetEntity;
```

Comment:
do not locate elements using text content.

Reply:
The above locators also cannot be changed as these are the links and unique identifiers for the above.